### DISCUSSION TITLE
not export special js file to html file under pages

### DISCUSSION THREAD

**Discussion by zitanlin** [vercel/next.js#25986] [Help]
How to not export special file to html file under pages. 

![image](https://user-images.githubusercontent.com/23631975/121541473-ea6eab00-ca39-11eb-8841-35e608ffd9d1.png)

Refer to the picture above：
I don't expect to compile a.js file. Because a.js file is not a page, but i have to put in this place.

**Comment by jacobhq** [comment]
Put your file in the `assets` directory, and it will be mapped to `/`.
